examples/attention_sink/example_mha_sink_fwd_bhsd_wgmma_pipelined.py

```
@@ -307,6 +307,29 @@ def main(
         print("Tilelang: {:.2f} TFlops".format(total_flops / latency * 1e-9))
 
 
+def benchmark(
+    batch: int = 1,
+    heads: int = 32,
+    seq_q: int = 256,
+    seq_kv: int = 256,
+    dim: int = 128,
+    window_size: Optional[int] = None,
+    dtype: str = "float16",
+    tune: bool = False,
```

Ruff (0.14.8)

Unused function argument: `tune`

(ARG001)